src/python/src/test/dist.py

```diff
@@ -2,13 +2,15 @@
 
 from ..lib.dist import kmerize_samples
 
+ids = ["1", "2"]
+samples = {}
+for id in ids:
+    samples[id] = ["".join(random.choice("ATGC") for _ in range(20))
+                   for _ in range(10)]
 
-samples = {
-    "1": ["".join(
-        random.choice("ATGC") for _ in range(100)) for _ in range(10)],
-    "2": ["".join(
-        random.choice("ATGC") for _ in range(100)) for _ in range(10)]
-}
 
+kmerized = kmerize_samples(samples, 10)
 
-print(kmerize_samples(samples, 10))
+for id in ids:
+    assert id in kmerized.keys()
+    assert len(kmerized[id]) > 0
```

Comment:
Я понял, где они терялись. Забыл склеить вывод с разных процессов reduce-ом. Мой тупняк. 

> On 20 May 2016, at 02:38, Nikolay Romashchenko <notifications@github.com> wrote:
> 
> In src/python/src/test/dist.py <https://github.com/nromashchenko/nir/pull/2#discussion_r63969629>:
> 
> >  
> > -print(kmerize_samples(samples, 10))
> > +for id in ids:
> > +    assert id in kmerized.keys()
> > +    assert len(kmerized[id]) > 0
> Твой тест возвращал что-то вроде [ {'2' : Counter(...) } ], хотя ожидаются каунтеры ко всем образцам. Такие простые ассерты не работали, т.к. где-то по пути каунтеры ко всем образцам, кроме последнего, терялись. Я вдумчиво смотрел на это, не понимал, в чем дело и в итоге переписал так, как ты наблюдаешь.
> 
> —
> You are receiving this because you were assigned.
> Reply to this email directly or view it on GitHub <https://github.com/nromashchenko/nir/pull/2/files/2d6fb7ed2f52484a99cb3cdd6a72613124b2ab20#r63969629>